whois-api/src/main/java/net/ripe/db/whois/api/rest/HttpsAPIKeyAuthCustomizer.java

```diff
@@ -0,0 +1,128 @@
+package net.ripe.db.whois.api.rest;
+
+import com.google.common.net.HttpHeaders;
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletRequestWrapper;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.ws.rs.core.UriBuilder;
+import net.ripe.db.whois.common.apiKey.ApiKeyAuthServiceClient;
+import net.ripe.db.whois.common.apiKey.ApiKeyUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.jetty.http.HttpStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.Objects;
+
+@Component
+public class HttpsAPIKeyAuthCustomizer implements Filter {
```

Comment:
Rename to `HttpsAPIKeyAuthFilter` to match interface

Reply:
Also rename `HttpsBasicAuthCustomizer` to `httpsBasicAuthFiler` has the same issue